pkg/obfuscator/cluster_id.go

```diff
@@ -0,0 +1,58 @@
+package obfuscator
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/openshift/must-gather-clean/pkg/schema"
+)
+
+var (
+	// From OCM codebase, pkg/models/uid.go
+	ocmClusterIdPattern            = "[0123456789abcdefghijklmnopqrstuv]{32}"
```

Comment:
So, this might produce false hits, i.e. when there is some random text value like a sha: `2b4a2379dafaef96fce39c29b7ab2b6424x`
That string will be redacted as well. 

Should I keep it to have some false hits or should I rather be more strict on matching?